skypy/galaxies/velocity_dispersion.py

```diff
@@ -0,0 +1,69 @@
+r"""Sample from velocity distribution function
+
+"""
+
+import numpy as np
+import scipy.special as special
+
+
+def schecter_vdf(vd_min, vd_max, resolution=100, size=None, scale=1.):
+    r"""Sample from velocity dispersion function of elliptical galaxies in the local universe [1]_.
+
+    Parameters
+    ----------
+    vd_min, vd_max: int
+        Lower and upper bounds of random variable x. Samples are drawn uniformly from bounds.
+    resolution: int
+        Resolution of the inverse transform sampling spline. Default is 100.
+    size: int
+        Number of samples returned. Default is 1.
+
+    Returns
+    -------
+    velocity_dispersion: array_like
+        Velocity dispersion drawn from Schechter function.
+
+    Warnings
+    --------
+    Inverse cumulative dispersion function is approximated from the function
+    using quadratic interpolation. The user should specify the resolution to
+    satisfy their numerical accuracy.
+
+        Notes
+    -----
+    The probability distribution function :math:`p(\sigma)` for redshift :math:`\sigma`
+    is given by Choi et al. [1]_ as
+
+    .. math::
+        \begin{eqnarray}&&{dn}={\phi }_{*}{(\displaystyle \frac{\sigma }{{\sigma }_{*}})}^{\alpha }
+        \mathrm{exp}[-{(\displaystyle \frac{\sigma }{{\sigma }_{*}})}^{\beta }]\displaystyle
+        \frac{\beta }{{\rm{\Gamma }}(\alpha /\beta )}\displaystyle \frac{1}{\sigma },
+        \end{eqnarray}\tag{3} \;.
```

Comment:
Can you make this shorter? It does not need to be so long. See [here](https://github.com/skypyproject/skypy/blob/df2536974c122dd72498f8d146913c14c832833b/skypy/galaxies/stellar_mass.py#L51).